http/handler.go

```diff
@@ -39,6 +41,9 @@ func Handler(core *vault.Core) http.Handler {
 	mux.Handle("/v1/sys/", handleLogical(core, true, nil))
 	mux.Handle("/v1/", handleLogical(core, false, nil))
 
+	// This is probably wrong in quite a few ways - unsure how to register it properly? Something like Mounting, maybe?
```

Comment:
Ah, okay. I definitely think some input from HashiCorp on this would be good to get. I agree auto mounting would be the way to go with it. But wouldn't be too sure on the way to implement it as I've not looked at the code in any depth.